repose-aggregator/components/services/http-client-service/http-client-service-api/src/main/resources/META-INF/schema/config/http-connection-pool.xsd

```diff
@@ -239,14 +239,22 @@
             </xs:annotation>
         </xs:attribute>
 
-        <xs:attribute name="chunked-encoding" type="xs:boolean" use="optional" default="true">
+        <xs:attribute name="chunked-encoding" type="chunkedEncodingType" use="optional" default="true">
             <xs:annotation>
                 <xs:documentation>
                     <html:p>
                         Determines whether or not this client should send chunked data when forwarding requests.
                     </html:p>
                     <html:p>
-                        WARNING: Setting this attribute will cause repose to attempt to retrieve the
+                        If set to true, chunked data will always be sent.
+                        If set to false, chunked data will never be sent.
+                        If set to auto, chunked data will only be sent if the original request to Repose was chunked.
+                        Note that the auto setting only has meaning for the default connection pool, and only when
+                        requests are forwarded to the origin service. A value of auto for non-default pools will
+                        behave identically to a value of true.
```

Comment:
1. Is chunked encoding currently supported on all outgoing connections or just to the origin service?  If it's just to the origin service (which is what I would expect now that you mention it), I wouldn't change that behavior in this story.  I would just document this fact well and add a comment in https://repose.atlassian.net/browse/REP-3589 to address it.  Otherwise, what is the current behavior for non-origin service requests?
2. If you're talking about changing the behavior of non-origin service requests, then yes, I'd agree with you on it being a breaking change in a bad way, and I wouldn't do it here.
3. To your third point, this might be a breaking config change if anyone was hipster enough to configure a "1" or "0" in this field which is technically allowed.  ಠ_ಠ